plugins/modules/aci_config_export_policy.py

```diff
@@ -0,0 +1,312 @@
+        }
+    }
+previous:
+  description: The original configuration from the APIC before the module has started
+  returned: info
+  type: list
+  sample:
+    [
+        {
+            "fvTenant": {
+                "attributes": {
+                    "descr": "Production",
+                    "dn": "uni/tn-production",
+                    "name": "production",
+                    "nameAlias": "",
+                    "ownerKey": "",
+                    "ownerTag": ""
+                }
+            }
+        }
+    ]
+proposed:
+  description: The assembled configuration from the user-provided parameters
+  returned: info
+  type: dict
+  sample:
+    {
+        "fvTenant": {
+            "attributes": {
+                "descr": "Production environment",
+                "name": "production"
+            }
+        }
+    }
+filter_string:
+  description: The filter string used for the request
+  returned: failure or debug
+  type: str
+  sample: ?rsp-prop-include=config-only
+method:
+  description: The HTTP method used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: POST
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(
+        name=dict(type="str"),
+        description=dict(type="str"),
+        format=dict(type="str", choices=["json", "xml"]),
+        target_dn=dict(type="str"),
+        snapshot=dict(type="bool"),
+        export_destination=dict(type="str"),
+        scheduler=dict(type="str"),
+        start_now=dict(type="bool"),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "present", ["name"]],
+            ["state", "absent", ["name"]],
+        ],
+    )
+
+    aci = ACIModule(module)
+
+    name = module.params.get("name")
+    description = module.params.get("description")
+    format = module.params.get("format")
+    target_dn = module.params.get("target_dn")
+    snapshot = aci.boolean(module.params.get("snapshot"))
+    export_destination = module.params.get("export_destination")
+    scheduler = module.params.get("scheduler")
+    start_now = aci.boolean(module.params.get("start_now"), "triggered", "untriggered")
```

Comment:
See start_now was added but do not see this reflected back in the test cases. Could you provide tests / asserts for both "triggered" and "untriggered" state?